src/app/modules/browse/browse-controller.js

```diff
@@ -140,9 +140,29 @@
             }
             return _localize(message, params);
         }
+        function setHiddenBooksMessages() {
+            if ($scope.allLicenses) {
```

Comment:
We don't have a count of restricted licenses available when showing them.  Should we?